src/modules/private-share-folder/private-sharing.controller.ts

```diff
@@ -239,4 +242,55 @@ export class PrivateSharingController {
         ),
     };
   }
+
+  @Delete('stop/folder-id/:folderId')
+  @ApiOperation({
+    summary: 'Stop sharing one folder',
+  })
+  @ApiBearerAuth()
+  async stopSharing(
+    @Param('folderId', ParseUUIDPipe) folderUuid: string,
+    @UserDecorator() user: User,
+  ): Promise<any> {
+    try {
+      return await this.privateSharingUseCase.stopSharing(folderUuid);
+    } catch (error) {
+      new Logger().error(
+        `[PRIVATESHARING/STOP] ERROR: ${
```

Comment:
You must catch the specific errors defined in the use case and handle them. This error for example:

https://github.com/internxt/drive-server-wip/blob/59fe6c7bbd4c8732691028b6491a26f714655c21/src/modules/private-share-folder/private-sharing.usecase.ts#L101

how to do it :
https://github.com/internxt/drive-server-wip/blob/59fe6c7bbd4c8732691028b6491a26f714655c21/src/modules/folder/folder.controller.ts#L304